### PR TITLE
integrate Dartino launching into dartlang framework

### PR DESCRIPTION
**Pull request by danrubel**
* integrated Dartino app launch into dartlang launching framework
* use 'dartino flash' command to compile and deploy rather than specialized Atom specific code
* show command line details in console during launch

@devoncarew

### CODE REVIEW

File: lib/dartino/device/device.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import 'dart:async';
+
+import 'package:atom_dartlang/atom.dart';
+
+import '../dartino_util.dart';
+import '../launch_dartino.dart';
+import '../sdk/dartino_sdk.dart';
+import 'stm32f746disco.dart';
```
Comment (devoncarew) [inline lib/dartino/device/device.dart]: Perhaps a slightly less exact but more user friendly file name?

**Comment by devoncarew** [comment]
lgtm

File: lib/dartino/device/device.dart (hunk 1 above)
Comment (danrubel) [inline lib/dartino/device/device.dart]: Point taken, but these are the names used in the dartino project, and I want to keep them the same in both atom and in dartino for my own sanity.

Comment (devoncarew) [inline lib/dartino/device/device.dart]: Makes sense -

Comment (danrubel) [inline lib/dartino/device/device.dart]: I'm hoping that after a few boards, I can generalize, and then yes, the names will be more user friendly at that point :-)

**Comment by danrubel** [comment]
Merged https://github.com/dart-atom/dartlang/commit/7fe56f9524082cd9c309531838535632176a92bd

**Pull request closed without merging**